src/app/header/login/login.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="row">
+    <div class="col-6 LoginRegText">
+      <h1>Login</h1>
+      <h2>Get access to your Orders, Wishlist and Recommendations</h2>
+    </div>
+    <div class="col-6">
+      <form>
+        <div class="form-group">
+          <label for="exampleInputEmail1" class="bmd-label-floating"
```

Comment:
Please take reference from my comments on the registration form

Reply:
done